Q-And-A/Q-And-A/ViewControllers/AnswerViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  AnswerViewController.swift
+//  Q-And-A
+//
+//  Created by Angel Buenrostro on 1/15/19.
+//  Copyright © 2019 Angel Buenrostro. All rights reserved.
+//
+
+import UIKit
+
+class AnswerViewController: UIViewController {
+    
+    var questionController: QuestionController? = nil
+    
+    var question: Question? = nil
```

Comment:
After passing data to `AnswerViewController` you need to check if it's value change. Then update views based. `didSet` is a good way to do that. For more: [article](https://www.hackingwithswift.com/read/8/5/property-observers-didset)
```suggestion
    var question: Question? {
        didSet { updateViews() }
    }
```